requirements-dev.in

```diff
@@ -11,3 +11,4 @@ pip-tools>=6.13.0
 asserts
 pytest-mock
 Flake8-pyproject
+moto[s3,sqs]
```

Comment:
Does this change relate to the other environment changes/ the Jira change? All good for it go in but might be good for the description to mention if its related or not

Reply:
It's because the version of utils we will go to next doesn't include this by default